src/event/handlers/GhostPingHandler.ts

```diff
@@ -10,6 +10,7 @@ class GhostPingHandler extends EventHandler {
 	async handle(message: Message): Promise<void> {
 		if (message.mentions.users.first() || message.mentions.roles.first()) {
 			if (!message.author?.bot) {
+				if (message.mentions.users.first()?.id === message.author.id && message.mentions.users.size === 1) return;

```

Comment:
Would be nice to shorten this line:
```suggestion
				const usersMentioned = message.mentions.users;
				
				if (usersMentioned.first()?.id === message.author.id && usersMentioned.size === 1) return;

```